ffv1.md

```diff
@@ -198,8 +196,23 @@ $$Q_{i}[a-b]=Table_{i}[(a-b)\&255]$$
 
 ## Colorspace
 
+FFV1 supports two colorspaces: YCbCr and JPEG2000-RCT. Both colorspaces allow an optional Alpha plane that can be used to code transparency data.
+
+### YCbCr
+
+In YCbCr colorspace, the Cb and Cr planes are optional, but if used then MUST be used together. Omitting the Cb and Cr planes codes the frames in grayscale without color data. An FFV1 frame using YCbCr MUST use one of the following arrangements:
+
+- Y
+- Y, Alpha
+- Y, Cb, Cr
+- Y, Cb, Cr, Alpha
+
+When FFV1 uses the YCbCr colorspace, the Y plane MUST be coded first. If the Cb and Cr planes are used then they MUST be coded after the Y plane. If an Alpha (transparency) plane is used, then it MUST be coded last.
+
 ### JPEG2000-RCT
 
+JPEG2000-RCT is a Reversible Color Transform that codes RGB (red, green, blue) planes losslessly in a modified YCbCr colorspace. Reversible conversions between YCbCr and RGB use the following formulae.
```

Comment:
I see, thank you! This must me the reason why many years we decided to work internally with a big LUT (BTW working in Gray code instead of regular binary permits to reduce the rounding errors). Then I change my mind and do **not** suggest to mention bijective or bijective function.